### ISSUE TITLE
doesn't work well in step 4

### ISSUE THREAD

**Issue by li-jing-wen** [laihuiyuan/Generic-resources-for-TST#5]
Hello, I run the code according to your instructions, but the loss in step4 seems to be unable to be reduced. The screenshot is as follows:
![屏幕截图 2022-12-31 161032](https://user-images.githubusercontent.com/88104213/210130121-55261e48-3faa-413d-9468-06cc6d49a351.png)

I have done Further Pre-training of Step 3 and the loss is 0.01. The Step 3 trained model was called in train_ibt.py, but the loss still could not be reduced. Is this a normal phenomenon?

**Comment by laihuiyuan** [comment]
Hi! I think this is normal. Can you try to use the checkpoint you have to perform inference and automatic evaluation?
